deny.toml

```diff
@@ -23,8 +23,10 @@ ignore = [
     "RUSTSEC-2022-0071",
     # ansi_term is Unmaintained (via structopt(0.3.26)).
     "RUSTSEC-2021-0139",
-    # proc-macro-error is Unmaintained (via structopt-derive(0.4.18)
+    # proc-macro-error is Unmaintained (via structopt-derive(0.4.18)).
```

Comment:
it might be worth considering `clap` instead of `structopt` if it brings in unmaintained crates.

Reply:
Raised #738